### ISSUE TITLE
how to use pretrained modles?

### ISSUE THREAD

**Issue by xingchen0120** [SamiraMalek/Weighted-Belief-Propagation#2]
Dear Author:
Hi, Could you give more details about how to use pretrained models?  For example, if I run MVN_63_45.py and get N29_63_45.npy, then How can I use this model to verify the accuracy of the network.

Best wishes.

**Comment by SamiraMalek** [comment]
Hi,
MVN_63_45.py is a code for training the neural network. It does not save any models and N29_63_45.npy is the BER during the training.
Best,
Samira

**Comment by xingchen0120** [comment]
Hi,
Thank you for the answer. If I run MVN_63_45.py to train the neural network, then how can I save the models and  test the accuracy of the network? 
Best,
Xingchen

**Comment by SamiraMalek** [comment]
Hi, This is only the code for training. At first, you should generate training data. Then, you could put your test data as input for the trained model with out saving the model.
